tools/pylint-extensions/azure-pylint-guidelines-checker/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## Version 0.0.9 (2023-06-26)
```

Comment:
I don't think we normally put "Version" in the header, so as long as no tool is parsing this it's probably okay